test/ng/directive/formSpec.js

```diff
@@ -577,6 +649,83 @@ describe('form', function() {
     expect(child.$error.required).toEqual([inputB]);
   });
 
+
+    it('should ignore changes in manually removed child forms', function() {
+      doc = $compile(
+        '<form name="myForm">' +
+          '<ng-form name="childform">' +
+            '<input name="childformcontrol" ng-maxlength="1" ng-model="value"/>' +
+          '</ng-form>' +
+        '</form>')(scope);
+
+      var form = scope.myForm;
+      var childformController = doc.find('ng-form').eq(0).controller('form');
```

Comment:
You know you could simply write `form.childform`, right ? :stuck_out_tongue: 
<sub>(It applies in all tests.)</sub>